### PR TITLE
S72.57 Add Tab, TabConnector for dashboard endpoints

### PR DESCRIPTION
**Pull request by ronoudgenoeg**

### CODE REVIEW

File: src/Connector/Tab/TabConnector.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     */
+    public function canDelete()
+    {
+        return false;
+    }
+
+    /**
+     * @return bool
+     */
+    public function canCreate()
```
Comment (mtricht) [inline src/Connector/Tab/TabConnector.php]: Everything is by default set to false, so you only have to override this method!

Comment (pvaass) [inline src/Connector/Tab/TabConnector.php]: Amazingness of our lib

Comment (ronoudgenoeg) [inline src/Connector/Tab/TabConnector.php]: @swordbeta @pvaass Done!

**Comment by pvaass** [comment]
Actually there's no reason for the POST, PUT and DELETE to be disabled, since the Klipfolio API supports this on Tabs anyway.

**Pull request merged**